### PR TITLE
Add housekeeping functions

### PR DESCRIPTION
**Pull request by atvanguard**

### CODE REVIEW

File: contracts/root/RootChain.sol [hunk 1]
```diff
@@ -131,4 +125,13 @@ contract RootChain is RootChainStorage, IRootChain {
         headerBlock.root = bytes32(dataList[3].toUintStrict());
         headerBlock.createdAt = now;
     }
+
+    // HOUSEKEEPING functions
+    function setNextHeaderBlock(uint256 _value) public onlyOwner {
+        _nextHeaderBlock = _value;
+    }
+
+    function setHeimdallId(string memory _heimdallId) public onlyOwner {
```
Comment (0xAshish) [inline contracts/root/RootChain.sol]: how are we setting/updating `registry`?

Comment (0xAshish) [reply contracts/root/RootChain.sol]: before it was in the constructor.

**Comment by 0xAshish** [comment]
@atvanguard this should go to stage-1 branch

File: contracts/root/RootChain.sol (hunk 1 above)
Comment (atvanguard) [reply contracts/root/RootChain.sol]: Missed committing `RootChainProxy`. Check now.

**Pull request merged**